lib/aggregate/base.rb

```diff
@@ -28,6 +28,14 @@ def <=>(other)
       end || 0
     end
 
+    def self.hashed_keys
```

Comment:
This name is super generic. I would have expected it to return data from the aggregate, not config data.